ablock.go

```diff
@@ -633,10 +632,19 @@ func GetABlock(keymr string) (ablock *ABlock, raw []byte, err error) {
 	return wrap.ABlock, raw, nil
 }
 
-// GetABlockByHeight requests an ABlock of a specific height from the factomd
-// API.
-func GetABlockByHeight(height int64) (ablock *ABlock, raw []byte, err error) {
-	params := heightRequest{Height: height}
+// GetABlock requests a specific ABlock from the factomd API with the raw data
+func GetABlock(keymr string) (ablock *ABlock, raw []byte, err error) {
+	return getABlock(keymr, false)
+}
+
+// GetSimpleABlock requests a specific ABlock from the factomd API without the raw data
+func GetSimpleABlock(keymr string) (ablock *ABlock, err error) {
```

Comment:
"Simple" doesn't convey clearly about what this method does or what is simple about it. We are going for a breaking change release (0.4) so I'd jump on the opportunity to break things to make them better (named)! I suggest:

- GetABlock(keymr string) (ablock *ABlock, err error)
- GetABlockWithRawData(keymr string) (ablock *ABlock, raw []byte, err error)

99% use case should not need the raw data, so that method should show up as the default choice for the developer. Note that if someone upgrade from 0.3.x to 0.4, the change of behavior of GetABlock will be made obvious because of the change in the return signature.

Reply:
See my reply above